asciidoctorj-api/src/main/java/org/asciidoctor/OptionsBuilder.java

```diff
@@ -59,15 +59,30 @@ public OptionsBuilder inPlace(boolean inPlace) {
         return this;
     }
 
+    /**
+     * Toggle including header and footer into the output.
+     *
+     * @param standalone <code>true</code> to generate a standalone output document
+     *                   (which includes the shell around the body content, such
+     *                   as the header and footer).
+     *                   Defaults to <code>true</code> when converting a file only,
+     *                   otherwise is <code>false</code>.
+     */
+    public OptionsBuilder standalone(boolean standalone) {
+        this.options.setStandalone(standalone);
+        return this;
+    }
+
     /**
      * Sets header footer attribute.
-     * 
-     * @param headerFooter
-     *            value.
+     *
+     * @param headerFooter value.
      * @return this instance.
+     * @deprecated Use {@link #standalone(boolean)} instead.
      */
+    @Deprecated
     public OptionsBuilder headerFooter(boolean headerFooter) {
-        this.options.setHeaderFooter(headerFooter);
+        this.options.setStandalone(headerFooter);
```

Comment:
Shouldn't we also keep this to `.setHeaderFooter()`?

Reply:
Indeed